apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1345,7 +1369,8 @@ get_pending_block() ->
 add_spend_txs() ->
     MineReward = rpc(aec_governance, block_mine_reward, []),
     MinFee = rpc(aec_governance, minimum_tx_fee, []),
-    MaxTxs = MineReward div (1 + MinFee),
+    MaxTxsInBlock = rpc(aec_governance, max_txs_in_block, []),
+    MaxTxs = min(MineReward div (1 + MinFee), MaxTxsInBlock), % so it can fit in one block
```

Comment:
(Not needed for this PR.) This 1 here is the same 1 as the amount (10 lines below), a variable would clarify.